math/wide_integer/uintwide_t.h

```diff
@@ -431,11 +437,13 @@
   using size_t    = std::uint32_t;
   using ptrdiff_t = std::int32_t;
 
+  // BEGINNOLINT
   static_assert((  (std::numeric_limits<size_t>::digits        >= 16)
                 && (std::numeric_limits<ptrdiff_t>::digits + 1 >= 16)),
                 "Error: size type and pointer difference type must be at least 16 bits in width (or wider)");
+  // ENDNOLINT
 
-  template<const size_t Width2> struct verify_power_of_two
+  template<const size_t Width2> struct verify_power_of_two // NOLINT(altera-struct-pack-align)
```

Comment:
I'm curious to know whether you're following the advice of these checks. They seem quite specific to the altera product. Then again, is the advice they give generally useful. (I think I just disable these in the config file.)

Reply:
To me it simply looked like the tool was wrong. It mentioned structure alignment of a convenience-memberless-compile-time-helper template structure. To me the whole thing seemed a bit rife for disabling, as I never fiddle with alignment anyway and try to write code that's independent of structure alignment.

I ended up explicitly `NOLINT`-ing it because I thought maybe in my quick assessment, i'm missing something and that comment serves to remind.